portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,249 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+/*body for all pages*/
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  margin: 0;
+}
+ 
+ /**************************** styling for the index.html file (main page) *************************/
+.header {
+  padding: 80px;
+  text-align: center;
+  background: #1abcbc;
```

Comment:
FYI, Google standard is to sort properties alphabetically

Reply:
Ok thank you! I will go through later and update this.